lib/safira/roulette/attendee_prize.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Safira.Roulette.AttendeePrize do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Safira.Accounts.Attendee
+  alias Safira.Roulette.Prize
+
+  schema "attendees_prizes" do
+    field :quantity, :integer
+
+    belongs_to :attendee, Attendee, foreign_key: :attendee_id, type: :binary_id
+    belongs_to :prize, Prize
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(attendee_prize, attrs) do
+    attendee_prize
+    |> cast(attrs, [:quantity, :attendee_id, :prize_id])
+    |> validate_required([:quantity, :attendee_id, :prize_id])
+    |> unique_constraint(:unique_attendee_prize)
+    |> validate_number(:quantity, greater_than: 0)
```

Comment:
Souldn't there be some sort of validation where the quantity can't be higher than a prize's `max_amount_per_attendee`?

Reply:
yes, I thought about that but I couldn't find any reasonable way to do it in the changeset, so the resolution will have to be associated with the logic before an update.

Reply:
Anyways the best solution would be do that verification here in the changeset.

Reply:
I add this validation in the changeset, and I think there is no problem to do so because `:max_amount_per_attendee` will not change. Nevertheless I think there should be a more elegant way to do this.